packages/docusaurus-plugin-openapi-docs/src/markdown/createRequestSchema.ts

```diff
@@ -520,13 +520,15 @@ function createDetailsNode(
   });
 }
 
-function createOneOfProperty(
+function createOneOfOrAnyOfProperty(
   name: string,
   schemaName: string,
   schema: SchemaObject,
   required: string[] | boolean,
   nullable: boolean | unknown
 ): any {
+  const type = schema.oneOf ? "oneOf" : "anyOf";
+  const children = schema[type] || [];
```

Comment:
I followed the precedent established in the above `createAnyOneOf` function, and I just made the minimum change to avoid the error being thrown. It looks like there is a ton of duplicated code in this file and between this file and `createResponseSchema`. I'd be happy to help refactor a bit, but wanted to do the smallest thing first, and see if you were interested in a larger refactor before embarking on one.